Xamarin.Forms.Platform.Android/AndroidTicker.cs

```diff
@@ -53,10 +53,10 @@ static bool IsAnimatorEnabled()
 			var resolver = global::Android.App.Application.Context?.ContentResolver;
 			if (resolver == null)
 			{
-				return false;
+				return true;
```

Comment:
To enable the animations we validate two points. On the one hand that the context and `ContentResolver` exists (are not null). I see this validation correct. On the other hand, we have the validation of ANIMATOR_DURATION_SCALE (more comments below).